gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/mapper/diagnosticreport/SpecimenMapperTest.java

```diff
@@ -162,6 +242,35 @@ private static DiagnosticReport generateDiagnosticReportWithNoSpecimenReference(
         return (DiagnosticReport) new DiagnosticReport().setId(DR_ID);
     }
 
+    private void assertMetaSecurityIsPresent(final Meta meta) {
+        final List<Coding> metaSecurity = meta.getSecurity();
+        final int metaSecuritySize = metaSecurity.size();
+        final Coding metaSecurityCoding = metaSecurity.get(0);
+        final UriType metaProfile = meta.getProfile().get(0);
+
+        assertAll(
+            () -> assertThat(metaSecuritySize).isEqualTo(1),
+            () -> assertThat(metaProfile.getValue()).isEqualTo(SPECIMEN_META_PROFILE),
+            () -> assertThat(metaSecurityCoding.getCode()).isEqualTo(NOPAT_CV.getCode()),
+            () -> assertThat(metaSecurityCoding.getDisplay()).isEqualTo(NOPAT_CV.getDisplayName()),
+            () -> assertThat(metaSecurityCoding.getSystem()).isEqualTo(NOPAT_CV.getCodeSystem()));
+    }
+
+    private RCMRMT030101UKCompoundStatement getCompoundStatement(RCMRMT030101UKEhrExtract ehrExtract) {
+        return ehrExtract.getComponent().get(0)
+            .getEhrFolder().getComponent().get(0)
+            .getEhrComposition().getComponent().get(0)
+            .getCompoundStatement().getComponent().get(0)
+            .getCompoundStatement();
+    }
```

Comment:
This method seems to be common between tests, would it be worth abstracting this out into the `TestUtility` class perhaps?

### ❓Questions:

* Will this method only work for these test files, and any test files with a similar structure? If so, is there any way we can make this dynamic to work for multiple use-cases?

Also, I would double-check your configured JDK version within IntelliJ, as I think if it was JDK 21 it would prompt you to use `.getFirst()` over `.get(0)`, worth checking out as this happened to me! 😄

Reply:
Yes, it is a valid point. Atm I am adding it as a duplication method/entry but later on I was actually planning to refactor all those methods all together and extract them from different classes related to Reduction. There are a lot of other duplicated functionality  between Reduction-related classes and tests.
Good suggestion about "getFirst" 👍